tests/unit/utils.test.ts

```diff
@@ -9,37 +9,30 @@ describe("Format functions", () => {
     expect(utils.formatLocationHash(hash)).toEqual(expected);
   });
 
+  test.each([
+    {
+      id: "rad:zKtT7DmF9H34KkvcKj9PHW19WzjT",
+      expected: "rad:zKtT7D…19WzjT",
+    },
+  ])("formatRepositoryId $id => $expected", ({ id, expected }) => {
+    expect(utils.formatRepositoryId(id)).toEqual(expected);
+  });
+
   test.each([
     {
       id: "did:key:z6MkmzRwg47UWQxczLLLFfkEwpBGitjzJ1vKPE8U9ymd6fz6",
+      kind: "node" as const,
       expected: "did:key:z6Mkmz…md6fz6",
     },
     {
       id: "z6MkmzRwg47UWQxczLLLFfkEwpBGitjzJ1vKPE8U9ymd6fz6",
+      kind: "node" as const,
```

Comment:
We should be able to remove this `kind` property from here right?

Reply:
e0cd45284c851e565ec0d9c81a638343c91c4251